sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.institutions.controller.ts

```diff
@@ -156,13 +157,13 @@ export class EducationProgramOfferingInstitutionsController extends BaseControll
   @HasLocationAccess("locationId")
   @ApiNotFoundResponse({
     description:
-      "Program to create the offering not found for the institution.",
+      "Program to create the offering not found for the institution or " +
+      "Not able to find the program in the provided location.",
```

Comment:
Minor, since it is part of the same sentence it should start with lower case to be correctly displayed on swagger.